packages/web-components/package.json

```
@@ -96,11 +98,16 @@
     "e2e:local": "node ./scripts/e2e.js --ui"
   },
   "devDependencies": {
+    "@custom-elements-manifest/analyzer": "^0.10.4",
```

Sorry I missed these (the github filters didn't show this file as one I owned) - these should be pinned here as well
```suggestion
    "@custom-elements-manifest/analyzer": "0.10.4",
```